README.md

```diff
@@ -336,6 +336,15 @@ Layout(
 
 ## Customising behaviour
 
+### Highlight required fields instead of optional ones
+
+If `TBXFORMS_HIGHLIGHT_REQUIRED_FIELDS=False` (or unset), optional fields will
+have "(optional)" appended to their labels. This is the default behaviour and
+recommended by GDS.
+
+If `TBXFORMS_HIGHLIGHT_REQUIRED_FIELDS=True`, required fields will have an
+asterisk appended to their labels and optional fields will not be highlighted.
+
```

Comment:
suggestion: highlight the marker classes (`tbxforms-field_marker--required`, `tbxforms-field_marker--optional`) should devs want to change their styling

Reply:
Good shout, done in https://github.com/torchbox/tbxforms/pull/78/commits/4fa218b6db64b90551397538a5f5f0b2a4584c7a